cocos/tween/actions/action-interval.ts

```diff
@@ -493,17 +497,20 @@ export class Repeat extends ActionInterval {
     private _actionInstant = false;
     private _innerAction: FiniteTimeAction | null = null;
 
-    constructor (action?: any, times?: any) {
+    constructor (action?: FiniteTimeAction, times?: number) {
         super();
-        times !== undefined && this.initWithAction(action as FiniteTimeAction, times as number);
+        this.initWithAction(action, times);
```

Comment:
Should check `times`?

Reply:
The checking logic has been moved to `initWithAction` since I think the constructor parameter should be matched with `initWithAction`.